src/main/features/response/form/models/howMuchDoYouOwe.ts

```diff
@@ -0,0 +1,39 @@
+import { IsDefined, IsPositive } from 'class-validator'
+import { Fractions } from '@hmcts/cmc-validators'
+import { ValidationErrors as DefaultValidationErrors } from 'forms/validation/validationErrors'
+import { toNumberOrUndefined } from 'shared/utils/numericUtils'
+
+export class ValidationErrors {
+  static readonly AMOUNT_NOT_VALID: string = 'Enter valid amount'
```

Comment:
isn't this a global validation message?

Reply:
It is, however the validation document i got had expressed different content. I checked it with ronan and he prefered the new one.
@timja-kainos

Reply:
Ive added a global variable with content that has been approved by Ronan.
Ive got @rattur to check to see if this content will apply for his work and if so, he will use this variable once we have merge this